tests/frameworks/gtest/include/gtest-api.h

```diff
@@ -97,13 +97,9 @@ std::vector<char> Format(NSString* message, Args ... args)
 }
 }
 
-namespace testing {
-namespace internal {
```

Comment:
also I'm pretty confused why this didn't work ...

---
In reply to: [72853566](https://github.com/Microsoft/WinObjC/pull/718#discussion_r72853566) [](ancestors = 72853566)